pkg/registry/csv.go

```diff
@@ -53,6 +60,49 @@ type ClusterServiceVersion struct {
 	Spec json.RawMessage `json:"spec"`
 }
 
+func NewFromBundleDirectory(bundleDir string) (*ClusterServiceVersion, error) {
+	files, err := ioutil.ReadDir(bundleDir)
+	if err != nil {
+		return nil, fmt.Errorf("error reading bundle directory %s, %v", bundleDir, err)
+	}
+
+	csvFile, err := func() (string, error) {
+		for _, f := range files {
```

Comment:
I think there are more clever ways we could find this file. For example we can check all the files in the directory to see if they're yaml files, and then for each yaml file attempt to serialize it into a CSV.

Otherwise this will fail if they don't follow the naming convention described here

Reply:
there is no hard requirement on the name of the file...